app/Jobs/SendPostToPhoenix.php

```diff
@@ -40,21 +41,22 @@ public function __construct(Post $post, $hasFile = true)
      *
      * @return void
      */
-    public function handle()
+    public function handle(Registrar $registrar)
     {
         $phoenix = new Phoenix;
+        $drupal_id = $registrar->find($this->post->signup->northstar_id)->drupal_id;
 
         // Data that every post will have
         $body = [
-            'uid' => 12345, // Grab drupal_id from northstar object?
+            'uid' => $drupal_id,
             'nid' => $this->post->signup->campaign_id,
-            'quantity' => $this->post->signup->quantity,
+            'quantity' => $this->post->signup->quantity_pending,
             'why_participated' => $this->post->signup->why_participated,
         ];
 
         // Data that everything except an update without a file will have
         if ($this->hasFile) {
-            $body['file_url'] = is_null($this->post->content->edited_file_url) ? $this->photo->file_url : $this->post->content->edited_file_url;
+            $body['file_url'] = is_null($this->post->content->edited_file_url) ? $this->post->file_url : $this->post->content->edited_file_url;
```

Comment:
@sbsmith86 @katiecrane found another small typo and figured I'd just put it in here instead of making a separate PR for it!